tardis/model/tests/test_base.py

```diff
@@ -202,3 +204,48 @@ def test_ascii_reader_exponential_law():
     for i, mdens in enumerate(expected_densites):
         assert_almost_equal(model.density[i].value, mdens)
         assert model.density[i].unit ==  u.Unit(expected_unit)
+
+###
+# Save and Load
+###
+
+
+@pytest.fixture(scope="module")
+def hdf_file_path(tmpdir_factory):
```

Comment:
You should define a top level fixture in `conftest.py` that can be used by all test modules. That way you only have to write this code once and if it needs changes, you only have to change one instance of it.